### PR TITLE
Feature/create item index page

### LINKED ISSUE #35: Merchant Items Index Page
As a merchant,
When I visit my merchant items index page ("merchant/merchant_id/items")
I see a list of the names of all of my items
And I do not see items for any other merchant

### PR DESCRIPTION
**Pull request by redferret**
### Features being added/updated
- Adds index page for a merchant's items
- New migration to add missing column `name` for items table

### Any issues this PR will unblock/block (Leave blank if no blockers are involved) 
- Will unblock PR #56 

- [x] Ready for merge (Don't merge unless the author of the PR has checked this)
- [x] All Tests are passing

(Link issue number next to `#` to automatically move the card on kanban board)

closes #35

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+  def edit
+  end
+
+  def update
+  end
+
+  private
 
+  def set_item
+    @item = Item.find(params[:id])
```
Comment (redferret) [inline app/controllers/items_controller.rb:27]: I prepared these for my other issues

File: app/views/items/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      <% @items.each do |item| %>
+        <tr id="item-<%=item.id%>">
+          <td><%=item.name%></td>
+          <td><%=item.unit_price%></td>
+          <td><%=item.quanity%></td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+</div>
```
Comment (redferret) [inline app/views/items/index.html.erb]: This makes lists look nice

File: db/migrate/20210602041505_add_column_name_to_items_table.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class AddColumnNameToItemsTable < ActiveRecord::Migration[5.2]
+  def change
+    change_table :items do |t|
+      t.string :name
+    end
+  end
+end
```
Comment (redferret) [inline db/migrate/20210602041505_add_column_name_to_items_table.rb:7]: Please rails db:migrate when you pull from main, I forgot the name column for items

**Review by JoannaCoPo — APPROVED** [review]
Looks good; will touch base at next stand up for full details

**Review by denverdevelopments — APPROVED** [review]

**Pull request merged**